src/models/thread.ts

```diff
@@ -20,6 +20,11 @@ import { MatrixEvent } from "./event";
 import { EventTimelineSet } from './event-timeline-set';
 import { Room } from './room';
 
+export enum THREAD_EVENTS {
+    ready = "Thread.ready",
+    update = "Thread.update"
```

Comment:
We normally use Camel Casing for our enum member naming

Reply:
Done.